src/SFA.DAS.AssessorService.Web/Views/Application/OptInConfirmation.cshtml

```diff
@@ -0,0 +1,32 @@
+@model SFA.DAS.AssessorService.Web.ViewModels.Apply.OptInConfirmationViewModel
+@{
+    ViewBag.Title = "Opt In Confirmation";
```

Comment:
To be in line with other confirmation pages, I think this should be "Opt in submitted" I think what i'll do is raise a ticket for Emma to run through the journey in an environment and take a look at the titles of the pages we've added